tools/RandomCodeGenerator.js

```diff
@@ -18,23 +19,6 @@ import Rust from "./utils/rust";
 import SQL from "./utils/sql";
 import Swift from "./utils/swift";
 
-export const Languages = {
```

Comment:
This was duplicated. So, I dunno if its removal was forgotten when it was moved into constants or not? @isabellabrookes ?

Reply:
I didn't put it back. I removed it. Then it got re-added.

Reply:
😂 might have even been my merge